include/freerdp/channels/cliprdr.h

```diff
@@ -116,14 +116,14 @@ typedef struct _CLIPRDR_HEADER CLIPRDR_HEADER;
 struct _CLIPRDR_CAPABILITY_SET
 {
 	UINT16 capabilitySetType;
-	UINT16 capabilitySetLength;
```

Comment:
I don´t like renaming symbols in public headers for no apparent reason.

Reply:
Then I'll revert. Just noticed that this is inconsistent with the documentation.